### PR TITLE
docs: make consistent, cleanup

### PR DESCRIPTION
**Pull request by uncenter**
Cleans up the READMEs to be clearer and consistent across tools.

### CODE REVIEW

File: whiskers/README.md [hunk 1]
```diff
@@ -5,42 +5,51 @@
   <img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
 </h3>
 
-<p align="center">
-  <a href="https://github.com/catppuccin/toolbox/stargazers"><img src="https://img.shields.io/github/stars/catppuccin/toolbox?colorA=363a4f&colorB=b7bdf8&style=for-the-badge"></a>
-  <a href="https://github.com/catppuccin/toolbox/issues"><img src="https://img.shields.io/github/issues/catppuccin/toolbox?colorA=363a4f&colorB=f5a97f&style=for-the-badge"></a>
-  <a href="https://github.com/catppuccin/toolbox/contributors"><img src="https://img.shields.io/github/contributors/catppuccin/toolbox?colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
-</p>
-
```
Comment (nekowinston) [inline whiskers/README.md:13]: these are all published to crates.io with their respective READMEs, i'd like if we keep these badges for each subdirectory

Comment (uncenter) [reply whiskers/README.md:13]: Ah, hadn't thought about that. Will revert.

**Comment by sgoudham** [comment]
I'm refactoring the whiskers in README in #111 so I'd appreciate if you could revert those changes for that here.

**Comment by uncenter** [comment]
> I'm refactoring the whiskers README in #111 so I'd appreciate if you could revert those changes for that here.

I like the usage of the table for installation instructions. Maybe I do that for each of the READMEs here and there wouldn't be a conflict after merging?

**Comment by sgoudham** [comment]
I think the table format only applies to `catwalk` and `puccinier` aside from `whiskers`, I don't mind if you update it for those but I'm planning on merging #111 probably right after this one (it's marinated long enough lol) so reverting the entire file would be best for me 👍

**Comment by nekowinston** [comment]
regarding the badges, we could still improve them by changing the issue button to https://github.com/catppuccin/toolbox/issues/new?labels=whiskers etc.
Note that puccinier doesn't have a label since it's effectively EOL

**Comment by uncenter** [comment]
> I think the table format only applies to `catwalk` and `puccinier` aside from `whiskers`

Aren't those... the only two other tools with READMEs?

**Comment by uncenter** [comment]
> regarding the badges, we could still improve them by changing the issue button to [catppuccin/toolbox/issues/new?labels=whiskers](https://github.com/catppuccin/toolbox/issues/new?labels=whiskers) etc. Note that puccinier doesn't have a label since it's effectively EOL

Will do... I'll just leave Puccinier as is I guess.

**Pull request closed without merging**